### DISCUSSION TITLE
durandal-vs-angular-a-real-world-comparison/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#48] [Blog comments]
# durandal-vs-angular-a-real-world-comparison/

https://blogs.taiga.nl/martijn/posts/durandal-vs-angular-a-real-world-comparison/

**Comment by disqus-comments-migrator** [comment]
<em>Obi commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 15 2014, 09:01</em>

---

<p>Nice article. As a (presently .Net) developer who had both options at the start of my SPA days a while ago and decided to go the DurandalJS route i can say you are right no every count. I think the major issue i have had with learning AngularJS is the same most people would say is it's strongest point and that is the sheer volume of resources. It is almost like there is no single source of truth and in order to get people to learn it, the examples are made simplistic and one is expected to refactor to support complexity or scale. DurandalJS however is DurandalJS, I have gone from the HelloWorld project on DurandalJS straight to production apps and I have not had to change much. With DurandalJS, there is ONE source of truth: <a href="http://www.durandaljs.com" rel="nofollow noopener" title="www.durandaljs.com">www.durandaljs.com</a>; it doesn't get sweeter than that!<br>I do have concerns about the longevity of DurandalJS because it's a one man show at the moment and especially because it's Kickstarter campaign wasn't so successful, however I think DurandalJS as a framework has a very bright future.<br>Bottom-line, like you stated Martijn, there are no winners; ultimately the winner is the developer who can take either one of these and turn it into a solid application that can make a difference and get them their paycheque!</p>

**Comment by disqus-comments-migrator** [comment]
<em>Lorenzo commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juni 04 2014, 09:57</em>

---

<p><a href="http://eisenbergeffect.bluespire.com/angular-and-durandal-converge/" rel="nofollow noopener" title="http://eisenbergeffect.bluespire.com/angular-and-durandal-converge/">http://eisenbergeffect.bluespire.com/angular-and-durandal-converge/</a></p>

**Comment by disqus-comments-migrator** [comment]
<em>Roberto Prevato commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 14 2014, 04:52</em>

---

<p>Hi,<br>I read with interest your article, and I appreciate the opinion of somebody who used both frameworks.<br>I am currently using Angular and only taken a brief glimpse to Durandal.</p><p>I would like to point out, that the initial sentence "<em>Angular is the safe choice for the future. I wouldn’t start any new projects based on Durandal 2.x anymore</em>"; is debatable, for the simple reason that the same thing can be said about the current Angular!</p><p>Since Angular 2.x will be radically different than the current one, saying that it is "the safe choice for the future" is just an assumption.</p>
